data-migration/sample-backup.yml

```diff
@@ -0,0 +1,34 @@
+# Sample Backup Definition File
+# Defines a set of volumes to be created in balenaOS 
+# For each volume supply a content definition consisting of a set of:
+#  - source - the file / directory to backup
+#  - target - target file / directory in volume 
+#    If no filter is given, source will be mapped under this name in the volume directory
+#    If a filter is given and source is a directory, a directory under this name is created in 
+#    the volume and the filtered content are mapped into that directory.
+#    If target is left away source is mapped to volume-name. This only makes sense for one source 
+#    per volume.
+#  - filter - filter sources by regex
+#    If filter expression is a valid regular exxpression and source is a directory
+#    all files in source that match the filter expression are mapped to the volume / target directory.
+
+- volume: 'test volume 1'
+  content:
+  - target: 'target dir 1.1'
+    source: /home/thomas/develop/balena.io/support
+  - source: '/home/thomas/develop/balena.io/customer/sonder/unitdata/UnitData files'
+    target: "target dir 1.2"
+- volume: 'test volume 2'
+  content:
+  - source: '/home/thomas/develop/balena.io/migrate/migratecfg/balena-migrate'
+    target: "target file 2.1"
+  - source: '/home/thomas/develop/balena.io/migrate/migratecfg/init-scripts'
+    target: 'target dir 2.2'
+    filter: 'balena-(c|i).*'
```

Comment:
What this filter supposed to match? In regex the `()` are groups, and not sure how groups would come in this filtering, without an example.

Reply:
Will make it clearer. For single characters [ci] would be the way to go, what I meant was rather 
'balena-(common|init)'